src/components/AdminPane/HOCs/WithComputedMetrics/WithComputedMetrics.js

```diff
@@ -25,14 +25,21 @@ export default function(WrappedComponent) {
 
     updateTotals = (actions, totalTasks, taskMetrics) => {
       _each(actions, (value, label) => {
-        taskMetrics[label] = _isNumber(taskMetrics[label]) ?
-                            taskMetrics[label] + value :
-                            value
-
-        const percentage = (1.0 * value / totalTasks) * 100.0
-        taskMetrics.percentages[label] =  _isNumber(taskMetrics.percentages[label]) ?
-                                          taskMetrics.percentages[label] + percentage :
-                                          percentage
+        if (label === "avgTimeSpent") {
+          taskMetrics.totalTimeSpent = _isNumber(taskMetrics.totalTimeSpent) ?
```

Comment:
Can you replace `_isNumber` with  `_isFinite`? `_isNumber` will return true for NaN whereas `_isFinite` will not, so we're trying to get away from `_isNumber` as it can lead to unexpected behavior.